backend/routes/forms/index.py

```diff
@@ -38,7 +38,7 @@ async def get(self, request: Request) -> JSONResponse:
             forms
         )
 
-    @requires(["authenticated", "admin"])
+    @requires(["authenticated", "Helpers"])
```

Comment:
I'm not sure how this works, and I can't seem to find a reference. Is this tied to a role helper somewhere?

Reply:
It isn't tied to a constant, but rather the new access system fetches all the user's roles, and adds their names as scopes.

Reply:
Can we move this to a constant at the top of the file or in the constants file so that it is easily changed?

Reply:
Does it need to be easily changed, or changed at all for that matter?